src/trackers/COMMON.py

```
@@ -551,55 +572,57 @@ async def parseCookieFile(self, cookiefile):
-        data = {}
+        params: dict[str, Any] = {}
+        data: dict[str, Any] = {}
         # get douban url
-        if int(meta.get('imdb_id')) != 0:
+        if int(meta.get('imdb_id', 0)) != 0:
             data['search'] = f"tt{meta['imdb_id']}"
-            ptgen = requests.get(url, params=data, timeout=30)
-            if ptgen.json()["error"] is not None:
+            ptgen_response = requests.get(url, params=data, timeout=30)
+            if ptgen_response.json()["error"] is not None:
                 for retry in range(ptgen_retry):
```

Ruff (0.14.10)

Loop control variable `retry` not used within loop body

Rename unused `retry` to `_retry`

(B007)